database/seeders/CourseSeeder.php

```diff
@@ -33,6 +33,11 @@ public function run(): void
                 'abbreviation' => 'WI',
                 'icon' => 'fa-sack-dollar',
                 'color' => 'rgb(22 101 52)',
+            ], [
+                'name' => 'B. Sc. Digital Innovation and Business',
+                'abbreviation' => 'DIB',
+                'icon' => 'fa-briefcase',
+                'color' => 'rgb(247 241 30)',
```

Comment:
Please use a colour that matches our current Tailwind theme. You can find an overview of all colours here: https://tailwindcss.com/docs/background-color#class-reference

Reply:
Edit: The colour should not yet be used for any course (no matter in which variation).

Reply:
See breaking change in https://github.com/fsr5-fhaachen/portals/pull/397 at the file https://github.com/fsr5-fhaachen/portals/blob/597a4cc5ab0c30cd5f47d8cf7144cc7a68b4009e/database/seeders/CourseSeeder.php